features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/CrashHandlerScenario.kt

```diff
@@ -22,9 +24,13 @@ internal class CrashHandlerScenario(
         val previousHandler = Thread.getDefaultUncaughtExceptionHandler()
 
         Thread.setDefaultUncaughtExceptionHandler({ t, e ->
```

Comment:
Also a (pre-existing) pet peeve is that the lambda can be taken out of the parentheses, which is the equivalent functionality but a simpler syntax:

```
Thread.setDefaultUncaughtExceptionHandler { t, e ->
    Log.d("Bugsnag", "Intercepted uncaught exception")
    previousHandler.uncaughtException(t, e)
}
```